services/src/main/java/org/openlearn/web/rest/PortfolioItemResource.java

```diff
@@ -129,9 +151,14 @@ public ResponseEntity update(@RequestBody @Valid final PortfolioItemDTO portfoli
 	@DeleteMapping(path = "/{id}")
 	@Secured({AuthoritiesConstants.ADMIN, AuthoritiesConstants.ORG_ADMIN, AuthoritiesConstants.INSTRUCTOR})
 	public ResponseEntity delete(@PathVariable final Long id) {
-		log.debug("DELETE request to delete portfolio item : {}", id);
-		portfolioItemService.delete(id);
-		return ResponseEntity.ok().build();
+		PortfolioItemDTO portfolioItem = portfolioItemService.findOne(id);
+		if (hasCreateUpdateDeleteAuthority(portfolioItem)) {
+			log.debug("DELETE request to delete portfolio item : {}", id);
+			portfolioItemService.delete(id);
+			return ResponseEntity.ok().build();
```

Comment:
200s (OKs) with no body should be 204(NO CONTENT)s

Reply:
Yup. Again, just a spacing change here (with the `if`). We'll probably just need to do a cleanup PR to handle all that.

Reply:
Because it's that way in every Resource controller